### PR TITLE
2pis

### PR DESCRIPTION
**Pull request by ishan190425**

### CODE REVIEW

File: donkeycar/memory.py
Comment (RohanAnand12) [inline donkeycar/memory.py:1]: What is type error as t L44

Comment (RohanAnand12) [inline donkeycar/memory.py:1]: add old and add new image what is that

File: donkeycar/parts/object_detector/stop_sign_detector_Tensorflow.py
Comment (RohanAnand12) [inline donkeycar/parts/object_detector/stop_sign_detector_Tensorflow.py:1]: Pls comment

File: donkeycar/parts/object_detector/stop_sign_detector_Tensorflow.py.save
Comment (RohanAnand12) [inline donkeycar/parts/object_detector/stop_sign_detector_Tensorflow.py.save:1]: .py.save?

**Review by RohanAnand12 — COMMENTED** [review]
Look at the comments i left for you. Pls try harder.

File: donkeycar/memory.py
Comment (ishan190425) [reply donkeycar/memory.py:1]: this was when stop sign needed unprocessed image

File: donkeycar/parts/object_detector/stop_sign_detector_Tensorflow.py.save
Comment (ishan190425) [reply donkeycar/parts/object_detector/stop_sign_detector_Tensorflow.py.save:1]: backup

File: donkeycar/memory.py
Comment (ishan190425) [reply donkeycar/memory.py:1]: Type error, is when the image is not read by the memory

**Review by RohanAnand12 — APPROVED** [review]
nice try!

**Pull request merged**